tools/vicon_arena_constructor/vicon_arena_constructor.cc

```diff
@@ -0,0 +1,219 @@
+
```

Comment:
I kinda feel that maybe this, it's boost dependency and need for the Vicon SDK would maybe be better off in another repo that just uses bob robotics like the snapshot bot or whatever.

Reply:
Yeah I think you're right actually. It is fairly self-contained.